index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+    <script src="./index.js" defer></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <link rel="stylesheet" href="./index.css">
+ 
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+<header>    <img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png" alt="">
+    <h1 id="review-app">Ghibli Review App</h1></header>
+   <main> <select name="" id="movie-titles">
+   <option value="" disabled selected></option> 
+</select>
+<section id="display-info">
+<h3 id="title"></h3>
+<p id="release-year"></p>
+<p id="film-description"></p>
+</section>
+<form action="">
+    <input id="user-review"type="text">
+    <input type="submit" id='subButton' value="Submit Review">
+    
+    
+    
+    </form>
+    <ul id="submitted-reviews"></ul>
```

Comment:
use prettier before submitting a PR, you don't want your tech lead reading code like this in the real world.